src/main/java/org/kainos/ea/DropwizardApplication.java

```diff
@@ -30,6 +32,7 @@ protected SwaggerBundleConfiguration getSwaggerBundleConfiguration(DropwizardCon
     public void run(final DropwizardConfiguration configuration,
                     final Environment environment) {
         // TODO: implement application
```

Comment:
Now that you have implemented you first controller you can remove this comment. Not a blocker tho :)